.eslintrc.cjs

```diff
@@ -262,12 +262,17 @@ const overrides = [
       '@typescript-eslint/no-non-null-assertion': 'off',
       'no-unused-vars': ['error', { 'args': 'all', 'argsIgnorePattern': '^_\\w+' }],
       'prefer-arrow/prefer-arrow-functions': 'warn',
-      'rulesdir/cactbot-matches-type-references': 'error',
       'rulesdir/cactbot-output-strings': 'error',
       'rulesdir/cactbot-response-default-severities': 'error',
       'rulesdir/cactbot-timeline-triggers': 'error',
     },
   },
+  {
+    'files': ['**/oopsyraidsy/data/**/*.ts', '**/raidboss/data/**/*.ts'],
```

Comment:
Thinking ahead!!

It would definitely be nice to sort out all the oopsyraidsy stuff, because I don't know that I can do config either until that's done.